src/Schema/Events/Tap.ts

```diff
@@ -292,6 +292,26 @@ export interface TappedConsign {
   subject: string
 }
 
+/**
+ * A user taps on an Artsy curated collection on a search tab
+ *
+ * This schema describes events sent to Segment from [[tappedCuratedCollection]]
+ *
+ * @example
+ * ```
+ * {
+ *   action: "tappedCuratedCollection",
+ *   context_module: "searchTab",
```

Comment:
**suggestion**: I'd make the following adjustments:

- Use the existing `search` screen owner type
- Use a context module for the curated collections section
- Add `destination_*` attributes to track the tapped collection
- Add a `position` attribute

```typescript
context_screen_owner_type: ScreenOwnerType.search
context_module: ContextModule.curatedCollections
destination_screen_owner_type: ScreenOwnerType.collection
destination_screen_owner_slug: string
destination_screen_owner_id: string
position: number
```

Reply:
thank you! I made changes you suggested, could you re-check?